From afb6542cd49f8188ec9857664f99a4e3444db98c Mon Sep 17 00:00:00 2001
From: Chia-I Wu <olvaffe@gmail.com>
Date: Thu, 3 Sep 2009 11:03:20 +0800
Subject: [PATCH 04/14] glapi: Prefix includes with glapi.

This allows different sets of generated headers to be used.
---
 src/mesa/glapi/glapi.c         |    8 ++++----
 src/mesa/glapi/glapi_getproc.c |    8 ++++----
 src/mesa/glapi/glthread.c      |    2 +-
 3 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/mesa/glapi/glapi.c b/src/mesa/glapi/glapi.c
index e36fccb..ad72fe4 100644
--- a/src/mesa/glapi/glapi.c
+++ b/src/mesa/glapi/glapi.c
@@ -69,9 +69,9 @@
 #include <assert.h>
 #endif
 
-#include "glapi.h"
-#include "glapioffsets.h"
-#include "glapitable.h"
+#include "glapi/glapi.h"
+#include "glapi/glapioffsets.h"
+#include "glapi/glapitable.h"
 
 
 /***** BEGIN NO-OP DISPATCH *****/
@@ -145,7 +145,7 @@ static GLint NoOpUnused(void)
    return 0;
 }
 
-#include "glapitemp.h"
+#include "glapi/glapitemp.h"
 
 /***** END NO-OP DISPATCH *****/
 
diff --git a/src/mesa/glapi/glapi_getproc.c b/src/mesa/glapi/glapi_getproc.c
index ed443c1..decdd05 100644
--- a/src/mesa/glapi/glapi_getproc.c
+++ b/src/mesa/glapi/glapi_getproc.c
@@ -34,9 +34,9 @@
 #include <string.h>
 #include "main/glheader.h"
 #include "main/compiler.h"
-#include "glapi.h"
-#include "glapioffsets.h"
-#include "glapitable.h"
+#include "glapi/glapi.h"
+#include "glapi/glapioffsets.h"
+#include "glapi/glapitable.h"
 
 
 static void
@@ -75,7 +75,7 @@ str_dup(const char *str)
 #endif
 
 /* The code in this file is auto-generated with Python */
-#include "glprocs.h"
+#include "glapi/glprocs.h"
 
 
 /**
diff --git a/src/mesa/glapi/glthread.c b/src/mesa/glapi/glthread.c
index 737fd4d..dc90b36 100644
--- a/src/mesa/glapi/glthread.c
+++ b/src/mesa/glapi/glthread.c
@@ -33,7 +33,7 @@
 #endif
 
 #include "main/compiler.h"
-#include "glthread.h"
+#include "glapi/glthread.h"
 
 
 /*
-- 
1.6.2.4

